src/metabase/models/interface.clj

```diff
@@ -216,6 +217,17 @@
   (or (mbql.normalize/normalize-fragment [:parameters] parameters)
       []))
 
+(defn- keywordize-temporal_units
+  [parameter]
+  (m/update-existing parameter :temporal_units (fn [units] (mapv keyword units))))
+
+(defn normalize-card-parameters-list
```

Comment:
This is not a good name. Enumerating the places where it's used is not good either.